iohyve

```diff
@@ -258,9 +258,25 @@ __get_zfs_pcidev_conf() {
 
 # List Guests
 __list() {
+	local flags="$1"
 	local pools="$(zfs list -H | grep iohyve | cut -d '/' -f 1 | uniq )"
 	(
-        printf "%s^%s^%s^%s^%s\n" "Guest" "VMM?" "Running" "rcboot?" "Description"
+	
```

Comment:
Using this method, the flags are al-la-carte allowing for flexibility and not using as much code as if you were to create an if statement or sub-sub-function for each desired output.